src/main/java/com/cburch/logisim/file/XmlReader.java

```diff
@@ -1195,11 +1195,13 @@ private void repairForWiringLibrary(Document doc, Element root) {
         labelMap,
         oldBaseLabel,
         newBaseLabel,
+        // FIXME: we should use _IDs here, right?
         "Poke Tool;" + "Edit Tool;Select Tool;Wiring Tool;Text Tool;Menu Tool;Text");
     addToLabelMap(
         labelMap,
         oldBaseLabel,
         wiringLabel,
+        // FIXME: we should use _IDs here, right?
```

Comment:
This seems sensible to me. @BFH-ktt1: What do you think?

Reply:
I think we do, but this also means that all other tools (i.e. "Poke Tool" etc) must be also amended to have its own `_ID` so all these strings here can be replaced by use of consts. I fix that later.